src/templates/common/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  transformBreadcrumbs,
+  deriveLastBreadcrumbFromPath,
+  deriveLcBreadcrumbs,
+  filterInvalidCrumbs,
+} from '@/lib/utils/breadcrumbUtils'
+import { BreadcrumbItem } from '@/types/dataTypes/drupal/field_type'
+
+interface BreadcrumbProps {
+  breadcrumbs?: BreadcrumbItem[]
+  breadcrumbTitle?: string
+  disableAnalytics?: boolean
+  deriveBreadcrumbsFromUrl?: boolean
+  replaceLastItem?: boolean
+  constructLcBreadcrumbs?: boolean
+  lcBreadcrumbsTitleInclude?: boolean
+  hideHomeBreadcrumb?: boolean
+  customCrumbHomeText?: string
+  entityPath?: string
+}
+const Breadcrumbs = ({
+  breadcrumbs,
+  entityPath,
+  breadcrumbTitle,
+  disableAnalytics,
+  deriveBreadcrumbsFromUrl,
+  replaceLastItem,
+  constructLcBreadcrumbs,
+  lcBreadcrumbsTitleInclude,
+  hideHomeBreadcrumb,
+  customCrumbHomeText,
+}: BreadcrumbProps) => {
+  if (!breadcrumbs) return null
+
+  if (customCrumbHomeText) {
+    breadcrumbs = breadcrumbs.map((crumb) => {
+      if (crumb.title === 'Home') {
+        return { ...crumb, title: customCrumbHomeText }
+      }
+      return crumb
+    })
+  } else if (hideHomeBreadcrumb) {
+    breadcrumbs = breadcrumbs.filter((crumb) => crumb.title !== 'Home')
+  }
+
+  if (deriveBreadcrumbsFromUrl) {
+    breadcrumbs = deriveLastBreadcrumbFromPath(
+      breadcrumbs,
+      breadcrumbTitle,
+      entityPath,
+      replaceLastItem
+    )
+  }
+
+  if (constructLcBreadcrumbs) {
+    breadcrumbs = deriveLcBreadcrumbs(
+      breadcrumbs,
+      breadcrumbTitle,
+      entityPath,
+      lcBreadcrumbsTitleInclude
+    )
+  }
+
+  const breadcrumbList = transformBreadcrumbs(breadcrumbs)
```

Comment:
**CONSIDER**
I wonder if this step belongs in the `formatter`.

Also, this step changes the shape of things. Do we want a type definition for a `BreadcrumbLink` or something to that effect?

Reply:
Agreed on the type definition, but I think that having that logic happen in one place here instead of every templates formatter is more DRY.  Happy to reconsider if we feel strongly about moving it though